### DISCUSSION TITLE
Getting error while installing

### DISCUSSION THREAD

**Discussion by nabindhami14** [prisma/prisma#24884] [Q&A]
### Bug description

# Getting error while installing prisma in Ubuntu22
```sh
nvm use 22
npm init -y
npm i express
cat package.json
```
```json
{
  "name": "app",
  "version": "1.0.0",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "description": "",
  "dependencies": {
    "express": "^4.19.2"
  }
}
```

```sh
npm i prisma
```

```sh
npm error code ECONNRESET
npm error errno ECONNRESET
npm error network Invalid response body while trying to fetch https://registry.npmjs.org/@prisma%2fdebug: aborted
npm error network This is a problem related to network connectivity.
npm error network In most cases you are behind a proxy or have bad network settings.
npm error network
npm error network If you are behind a proxy, please make sure that the
npm error network 'proxy' config is set properly.  See: 'npm help config'
npm error A complete log of this run can be found in: /home/nabin/.npm/_logs/2024-07-25T05_06_47_667Z-debug-0.log
```

> I have tried to switch from npm to bun, node versions 22, 20, 18 and cleared npm cache still getting same error. Installing other packages is working fine `@prisma/client`, `express`...

### How to reproduce

```sh
nvm use 22
npm init -y
npm i express
cat package.json

npm i prisma
```

### Prisma information

```sh
npm i prisma
```

### Environment & setup

- OS: Ubuntu22
- Node.js version: 22,20,18

### Prisma Version

```
latest
```

**Comment by ludralph** [comment]
Hi @nabindhami14 👋 

I can't seem to reproduce this error as I can install Prisma on my end. 

Given that other modules are installed correctly, can you try using `yarn` or maybe changing networks if possible?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
